modules/jaeger-install-overview.adoc

```diff
@@ -0,0 +1,19 @@
+////
+This CONCEPT module included in the following assemblies:
+- rhbjaeger-installation.adoc
+////
+
+[id="jaeger-install-overview_{context}"]
+= Jaeger installation overview
+
+The steps for installing {ProductName} are as follows:
+
+* Review the documentation and determine your deployment strategy.
+
+* If your deployment strategy requires persistent storage, install the Elasticsearch Operator via the OperatorHub.
+
+* Install the Jaeger Operator via the OperatorHub.
```

Comment:
Do we need to use "Build of Jaeger" Operator name? We need to decide what should show up in the OperatorHub card for the product.

Reply:
I'm trying to keep the naming as simple as possible in the documentation so that I can reuse it with Service Mesh.  The actual installation instructions say to "Click the *Jaeger Operator* provided by Red Hat", so I think which ever name we go with on the Operator it should be OK.